crypto/Makefile

```diff
@@ -50,11 +77,21 @@ ZKP_CFLAGS = \
 ZKP_PATH = ../vendor/secp256k1-zkp
 CFLAGS += -DSECP256K1_CONTEXT_SIZE=208
 
+# TODO remove this workaround once possible
+ifeq ($(CLANG_AT_LEAST_14),true)
+$(warning "warning: suppressing clang-14 compiler warning for secp256k1-zkp code")
+ZKP_CFLAGS += -Wno-bitwise-instead-of-logical
+endif
+
 VALGRIND ?= 1
 ifeq ($(VALGRIND),1)
 CFLAGS += -DVALGRIND
 endif
 
+ifeq ($(VALGRIND),1)
+CFLAGS += -DVALGRIND
+endif
+
```

Comment:
This seems to be a duplicate.

Reply:
You're right, fixed by changing the last commit.